src/rpc/register.h

```diff
@@ -50,9 +52,12 @@ static inline void RegisterAllCoreRPCCommands(CRPCTable &tableRPC)
 #ifdef ENABLE_EXODUS
     if (isExodusEnabled()) {
         RegisterExodusDataRetrievalRPCCommands(tableRPC);
-        RegisterExodusTransactionCreationRPCCommands(tableRPC);
         RegisterExodusPayloadCreationRPCCommands(tableRPC);
         RegisterExodusRawTransactionRPCCommands(tableRPC);
+
+        #ifdef ENABLE_WALLET
```

Comment:
Wrong indentation. All pre-processor directives should start at the first column (like line number 52 on the above).